components/vault/use-vault-configurator.js

```diff
@@ -110,7 +121,7 @@ export function useVaultConfigurator ({ onVaultKeySet, beforeDisconnectVault } =
       console.error('error setting vault key', e)
       toaster.danger(e.message)
     }
-  }, [getVaultEntries, updateVaultKey, set, get, remove, onVaultKeySet, keyReactiveVar])
+  }, [getVaultEntries, updateVaultKey, set, get, remove, onVaultKeySet, keyReactiveVar, me.id])
```

Comment:
```suggestion
  }, [getVaultEntries, updateVaultKey, set, get, remove, onVaultKeySet, keyReactiveVar, me?.id])
```

else :boom: without session